types/handsontable/index.d.ts

```diff
@@ -492,28 +504,19 @@ declare namespace ht {
     type DropdownColumn = AutocompleteColumn;
 }
 
-declare var Handsontable: {
-    new (element: Element, options: ht.Options): ht.Methods;
-    plugins: ht.Plugins;
-    hooks: ht.Hooks;
-    Dom: ht.Dom;
-    dom: ht.Dom;
-    helper: ht.Helper;
-    utils: ht.Utils;
-    renderers: ht.Renderers;
-    editors: ht.Editors;
-};
+declare class Handsontable extends __Handsontable.Core {
+    static plugins: Handsontable.Plugins;
+    static hooks: Handsontable.Hooks;
+    static Dom: Handsontable.Dom;
+    static dom: Handsontable.Dom;
+    static helper: Handsontable.Helper;
+    static utils: Handsontable.Utils;
+    static renderers: Handsontable.Renderers;
+    static editors: Handsontable.Editors;
+}
 
 declare module "handsontable" {
-    const Handsontable: {
-      new (element: Element, options: ht.Options): ht.Methods;
-      plugins: ht.Plugins;
-      hooks: ht.Hooks;
-      Dom: ht.Dom;
-      dom: ht.Dom;
-      helper: ht.Helper;
-      utils: ht.Utils;
-      renderers: ht.Renderers;
-      editors: ht.Editors;
-    };
+    export = Handsontable;
 }
+// export = Handsontable;
+// export as namespace Handsontable;
```

Comment:
I added these for someone else to try and test. The types are never resolved for me without using the `declare module "handsontable" { }`.